File: src/mysqlcookbook/recipes/api/03_library/harness2.py
#!/usr/bin/python3
# harness2.py: test harness for cookbook.py library

# Does not catch exceptions, so this script simply dies if
# a connect error occurs.

import cookbook

#@ _FRAG_
conn = cookbook.connect()
print("Connected")
conn.close()
print("Disconnected")
#@ _FRAG_

File: src/mysqlcookbook/recipes/api/03_library/harness2.rb
#!/usr/bin/ruby
# harness2.rb: test harness for Cookbook.rb library

# Does not catch exceptions, so this script simply dies if
# a connect error occurs.

require "Cookbook"

#@ _FRAG_
client = Cookbook.connect
print "Connected\n"
client.close
print "Disconnected\n"
#@ _FRAG_
